### PR TITLE
Add webfonts package

### PR DESCRIPTION
**Pull request by piofinn**
## 📥 Proposed changes

Add webfont files, and the means to use them in projects. Implements the new font in all components and in the portal. **NB!** The font sizes and line heights should be tweaked to match the new font.

## ☑️ Submission checklist

-   [x] I have read the [CONTRIBUTING](https://github.com/fremtind/jokul/blob/master/CONTRIBUTING.md) doc
-   [x] `yarn build` works locally with my changes
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] I have added necessary documentation (if appropriate)

## 💬 Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did, what alternatives you considered, etc...

### CODE REVIEW

File: packages/core/normalize.scss [hunk 1]
```diff
@@ -42,6 +42,19 @@ h1 {
...
+ * Set font-weight to bold on heading elements
+ */
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+    font-weight: bold;
```
Comment (lfbergee) [inline packages/core/normalize.scss]: Denne vil kunne påvirke utenfor seg selv, men vi ser kanskje ikke på det som et problem?

File: packages/select-react/example/index.tsx [hunk 2]
```diff
@@ -10,7 +10,10 @@ import "./index.scss";
 initTabListener();
 const SelectDemo = () => {
     const items = ["The flower shop", "I have cancer", "Throwing the football", "ChirpChirpChirp"];
-    const valuePairs = [{ value: "firstvalue", label: "Value 1" }, { value: "secondvalue", label: "Value 2" }];
+    const valuePairs = [
```
Comment (lfbergee) [inline packages/select-react/example/index.tsx]: pirk pirk, men foretrekker at endringer som ikke er relatert til endringen draes ut i en egen pr.

Comment (piofinn) [reply packages/select-react/example/index.tsx]: Enig, men den ville ikke gjennom linting uten denne 😅

File: packages/core/normalize.scss (hunk 1 above)
Comment (piofinn) [reply packages/core/normalize.scss]: Denne kan jeg ta vekk igjen. Fikk en rar bug i Firefox som seff viste seg å komme av caching... Bra catch!

File: packages/select-react/example/index.tsx (hunk 2 above)
Comment (lfbergee) [reply packages/select-react/example/index.tsx]: --no-verify

**Pull request merged**